src/ChatSettings.module.css

```diff
@@ -188,3 +188,260 @@
 .promptContainer::-webkit-scrollbar-thumb:hover {
   background: var(--colors-borderPrimary);
 }
+
+/* Toggle section styling */
+.toggleSection {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: var(--spacing-sm) 0;
+}
+
+/* Textarea styling for custom prompt */
+.promptTextarea {
+  color: var(--colors-textSecondary);
+  font-size: 1.2rem;
+  line-height: 1.6;
+  font-family: 'Monaco', 'Menlo', 'Ubuntu Mono', monospace;
+  white-space: pre-wrap;
+  word-wrap: break-word;
+  margin: 0;
+  background: var(--colors-bgSecondary);
+  border: 1px solid var(--colors-borderSecondary);
+  border-radius: var(--borderRadius-sm);
+  padding: var(--spacing-md);
+  width: 100%;
+  resize: vertical;
+  min-height: 300px;
+  outline: none;
+  transition: border-color 0.2s ease-in-out;
+}
+
+.promptTextarea:focus {
+  border-color: var(--colors-accent);
+  box-shadow: 0 0 0 2px rgba(var(--colors-accent-rgb), 0.1);
+}
+
+.promptTextarea::placeholder {
+  color: var(--colors-textTertiary);
+  opacity: 0.7;
+}
+
+/* Scrollbar styling for textarea */
+.promptTextarea::-webkit-scrollbar {
+  width: 8px;
```

Comment:
not a blocker, but we should have variables that can replace the need to use pixels. The should be defined here: https://github.com/Kava-Labs/hard-ai/blob/main/src/theme/themes.ts

usage: `padding: var(--spacing-sm);`